test/services/auth.test.ts

```diff
@@ -2,59 +2,128 @@
  * @jest-environment node
  */
 
-import { authenticateLogin } from '../../src/services/auth';
+import { User, AuthProviders } from '@prisma/client';
+
+import { authenticateLogin, authenticateGoogleLogin } from '../../src/services/auth/auth';
 import { user } from '../factories/user';
-import { createUser } from '../../src/app/actions/user';
+import { createUser } from '@/app/actions/user';
+import * as userRepository from '@/repositories/user';
+import * as identityRepository from '@/repositories/identity';
 
 describe('Auth:', () => {
-  describe('given correct credentials', () => {
-    it('returns user', async () => {
-      const userPayload = { name: user.name, email: user.email, password: user.password };
-
-      await createUser(userPayload);
-      const data = await authenticateLogin({
-        email: user.email,
-        password: user.password,
+  describe('Authenticate credential login', () => {
```

Comment:
Inconsistency with the other tests. Prefer to use the method name:

```suggestion
  describe('authenticateLogin', () => {
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/84/commits/f3a388edb96ff807b910abc71110e3dcca753c4b